recipes/swig/all/test_package/conanfile.py

```diff
@@ -5,13 +5,28 @@ class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     generators = "cmake", "cmake_find_package"
 
+    __cmake = None
+
+    @property
+    def _cmake(self):
+        if not self.__cmake:
+            self.__cmake = CMake(self)
+        return self.__cmake
+
+    @property
+    def _can_build(self):
+        # FIXEM: Fix SWIG Python build with Visual Studio
+        # Have a problem linking final module with Visual Studio. CMake finds
+        # correct Python library, but linker tries to use _d variant and fails.
+        return self.settings.compiler != "Visual Studio"
```

Comment:
Maybe just block debug?

Reply:
There are two issues I couldn't figure out with Visual Studio
1. Debug build keeps looking for _d even if I use PYTHON_LIBRARY_RELEASE
2. Target module files are relocated to Debug/Release subdirectories and Python is not able to find them

Well, I don't have Windows platform at hand to make this work, unfortunately.

Reply:
That's acceptable